src/main/java/games/strategy/triplea/delegate/MustFightBattle.java

```diff
@@ -819,6 +819,19 @@ private void pushFightLoopOnStack(final boolean firstRun) {
 
       @Override
       public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
+        if (Match.someMatch(m_attackingUnits, Matches.unitHasAttackValueOfAtLeast(1))) {
```

Comment:
Would that ever happen? I would imagine that the approach for defense units would work so I'll apply that here.